test/controllers/api/audio_files_controller_test.rb

```diff
@@ -49,4 +49,20 @@
     get :index, api_request_opts(story_id: story.id)
     assert_response :success
   end
+
+  describe '#original' do
+
+    before { @controller.current_user = user }
```

Comment:
I'd recommend rebasing against master to see how we need to do this now that tokens are being honored.